src/app/kyb/page.tsx

```
@@ -0,0 +1,367 @@
+    additionalInfoConsent: false,
+    privacyPolicyConsent: false,
+  });
+
+  const updateBusinessData = (updates: Partial<BusinessData>) => {
+    setBusinessData(prev => ({ ...prev, ...updates }));
+  };
+
+  const validateCurrentStep = (): boolean => {
+    switch (currentStep) {
+      case 1: // Business Information
+        const requiredBusinessFields = ['legalCompanyName', 'entityType', 'registrationNumber', 'email'];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)